README.md

```diff
@@ -82,10 +95,12 @@ This is an open source project which is officially supported by Exasol. For any
 
 ## Information for Users
 
-Virtual Schemas support 16 different SQL dialects, from OpenSource databases like PostgreSQL to commercial products like Oracle. 
-Please refer to the user-guide for a [full list of supported SQL dialects](doc/user-guide/user_guide.md#list-of-supported-dialects)
+* [Getting started with Virtual Schemas](https://docs.exasol.com/database_concepts/virtual_schemas.htm)
+* [List of Supported Dialect](doc/user-guide/dialects.md)
 
-* [User Guide](doc/user-guide/user_guide.md)
+Additional resources:
+* [Adapter Properties Reference](doc/user-guide/adapter_properties.md)
```

Comment:
Let's rename it to `Virtual Schema's Properties` and rename the file itself too. Adapter sounds too misleading